web-auth/src/components/EmailPassForm.svelte

```diff
@@ -68,7 +68,7 @@
 
 <div>
   {#if showForm}
-    <div class="mt-6 mb-4 flex flex-col gap-y-4" transition:slide>
+    <div class="mt-6 mb-4 flex flex-col gap-y-4" transition:slide|global>
```

Comment:
We have seen issues with global transitions. Has it been fixed in svelte4? Applies to all places where this change has been made.

Reply:
In Svelte 4, [transitions are local by default](https://svelte.dev/docs/v4-migration-guide#transitions-are-local-by-default). In Svelte 3, they were global by default. So this code ensures our application behaves the same as it did with Svelte 3.

All the changes to transitions were made automatically when I ran the Svelte 4 migration script: `npx svelte-migrate@latest svelte-4`.

I agree we should be wary of global transitions, but let's consider application behavior changes in a separate PR.